### ISSUE TITLE
can't install 

### ISSUE THREAD

**Issue by Aqualie** [LinusU/node-generate-rsa-keypair#2]
```
[ffsync@xxxx ~]$ git clone https://github.com/LinusU/node-generate-rsa-keypair
[ffsync@xxxx ~]$ cd node-generate-rsa-keypair/
[ffsync@xxxx node-generate-rsa-keypair]$ npm i
-
> generate-rsa-keypair@0.1.0 install /opt/fxa/node-generate-rsa-keypair
> node-gyp rebuild

make: Entering directory '/opt/fxa/node-generate-rsa-keypair/build'
  CXX(target) Release/obj.target/generate-rsa-keypair/binding.o
In file included from ../binding.cc:2:0:
../node_modules/nan/nan.h:324:27: error: redefinition of â€˜template<class T> v8::Local<T> Nan::imp::NanEnsureHandleOrPersistent(const v8::Local<T>&)â€™
   NAN_INLINE v8::Local<T> NanEnsureHandleOrPersistent(const v8::Local<T> &val) {
                           ^
../node_modules/nan/nan.h:319:17: note: â€˜template<class T> v8::Handle<T> Nan::imp::NanEnsureHandleOrPersistent(v8::Handle<T>&)â€™ previously declared here
   v8::Handle<T> NanEnsureHandleOrPersistent(const v8::Handle<T> &val) {
                 ^
../node_modules/nan/nan.h:344:27: error: redefinition of â€˜template<class T> v8::Local<T> Nan::imp::NanEnsureLocal(v8::Handle<T>&)â€™
   NAN_INLINE v8::Local<T> NanEnsureLocal(const v8::Handle<T> &val) {
                           ^
../node_modules/nan/nan.h:334:27: note: â€˜template<class T> v8::Local<T> Nan::imp::NanEnsureLocal(const v8::Local<T>&)â€™ previously declared here
   NAN_INLINE v8::Local<T> NanEnsureLocal(const v8::Local<T> &val) {
                           ^
../node_modules/nan/nan.h:757:13: error: â€˜node::smallocâ€™ has not been declared
     , node::smalloc::FreeCallback callback
             ^
../node_modules/nan/nan.h:757:35: error: expected â€˜,â€™ or â€˜...â€™ before â€˜callbackâ€™
     , node::smalloc::FreeCallback callback
                                   ^
../node_modules/nan/nan.h: In function â€˜v8::Local<v8::Object> NanNewBufferHandle(char*, size_t, int)â€™:
../node_modules/nan/nan.h:761:50: error: â€˜callbackâ€™ was not declared in this scope
         v8::Isolate::GetCurrent(), data, length, callback, hint);
                                                  ^
../node_modules/nan/nan.h:761:60: error: â€˜hintâ€™ was not declared in this scope
         v8::Isolate::GetCurrent(), data, length, callback, hint);
                                                            ^
../node_modules/nan/nan.h: In function â€˜v8::Local<v8::Object> NanNewBufferHandle(const char*, uint32_t)â€™:
../node_modules/nan/nan.h:768:67: error: no matching function for call to â€˜New(v8::Isolate*, const char*&, uint32_t&)â€™
     return node::Buffer::New(v8::Isolate::GetCurrent(), data, size);
                                                                   ^
In file included from ../node_modules/nan/nan.h:25:0,
                 from ../binding.cc:2:
/opt/fxa/.node-gyp/4.1.2/include/node/node_buffer.h:31:40: note: candidate: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, v8::Local<v8::String>, node::encoding) <near match>
 NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                        ^
/opt/fxa/.node-gyp/4.1.2/include/node/node_buffer.h:31:40: note:   conversion of argument 3 would be ill-formed:
In file included from ../binding.cc:2:0:
../node_modules/nan/nan.h:768:67: error: invalid conversion from â€˜uint32_t {aka unsigned int}â€™ to â€˜node::encodingâ€™ [-fpermissive]
     return node::Buffer::New(v8::Isolate::GetCurrent(), data, size);
                                                                   ^
In file included from ../node_modules/nan/nan.h:25:0,
                 from ../binding.cc:2:
/opt/fxa/.node-gyp/4.1.2/include/node/node_buffer.h:43:40: note: candidate: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, char*, size_t) <near match>
 NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                        ^
/opt/fxa/.node-gyp/4.1.2/include/node/node_buffer.h:43:40: note:   conversion of argument 2 would be ill-formed:
In file included from ../binding.cc:2:0:
../node_modules/nan/nan.h:768:67: error: invalid conversion from â€˜const char*â€™ to â€˜char*â€™ [-fpermissive]
     return node::Buffer::New(v8::Isolate::GetCurrent(), data, size);
                                                                   ^
../node_modules/nan/nan.h: In function â€˜v8::Local<v8::Object> NanNewBufferHandle(uint32_t)â€™:
../node_modules/nan/nan.h:772:29: error: could not convert â€˜node::Buffer::New(v8::Isolate::GetCurrent(), ((size_t)size))â€™ from â€˜v8::MaybeLocal<v8::Object>â€™ to â€˜v8::Local<v8::Object>â€™
     return node::Buffer::New(v8::Isolate::GetCurrent(), size);
                             ^
../node_modules/nan/nan.h: In function â€˜v8::Local<v8::Object> NanBufferUse(char*, uint32_t)â€™:
../node_modules/nan/nan.h:779:12: error: â€˜Useâ€™ is not a member of â€˜node::Bufferâ€™
     return node::Buffer::Use(v8::Isolate::GetCurrent(), data, size);
            ^
generate-rsa-keypair.target.mk:92: recipe for target 'Release/obj.target/generate-rsa-keypair/binding.o' failed
make: *** [Release/obj.target/generate-rsa-keypair/binding.o] Error 1
make: Leaving directory '/opt/fxa/node-generate-rsa-keypair/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/opt/node/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 4.2.3-1-ARCH
gyp ERR! command "/opt/node/bin/node" "/opt/node/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /opt/fxa/node-generate-rsa-keypair
gyp ERR! node -v v4.1.2
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok 

npm ERR! Linux 4.2.3-1-ARCH
npm ERR! argv "/opt/node/bin/node" "/opt/node/bin/npm" "i"
npm ERR! node v4.1.2
npm ERR! npm  v2.14.4
npm ERR! code ELIFECYCLE
npm ERR! generate-rsa-keypair@0.1.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the generate-rsa-keypair@0.1.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the generate-rsa-keypair package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls generate-rsa-keypair
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /opt/fxa/node-generate-rsa-keypair/npm-debug.log
```

**Comment by LinusU** [comment]
We need to upgrade to Nan 2.x for Node 4.x support

**Comment by LinusU** [comment]
Should be fixed by #3, released as `0.1.1` on npm.
